### DISCUSSION TITLE
I'm the guy that made the 4.2 dev release for reddit. Some devs seem to be unhappy about that.

### DISCUSSION THREAD

**Discussion by kroegerama** [OnionUI/Onion#737] [General]
As the title says. I made a dev release and one developer told me you might not like that. I really like your work and want to know how I should go on from now.

Should I add a full disclaimer that no one will provide support for the build? Or should I rather make my fork completely private so that no one has access anymore?

Thread: https://www.reddit.com/r/MiyooMini/comments/127jty1/onionos_42_beta_unofficial_builds_with_miyoo_mini/?sort=new

My explanation: https://www.reddit.com/r/MiyooMini/comments/127jty1/comment/jeezj7k/

**Comment by schmurtzm** [comment]
Some devs ? I don't understand why you post here, I'm one of the main dev of Onion and my opinion is the same here or on reddit...

**Reply by kroegerama** [reply]
Sorry, I did not know you are the main dev. I'll delete my fork.

**Reply by schmurtzm** [reply]
Anyway, I guess we can't fight against impatience indefinitely 🤷🏻‍♂️
At least indicate to put feedback on our issues repo or on [RGH discord server](https://discord.gg/retro-game-handhelds-529983248114122762) (in Miyoo -> miyoo-mini -> [Takes a village to grow an Onion](https://discord.com/channels/529983248114122762/931367023588569180)))

**Reply by kroegerama** [reply]
Yeah, I lost this fight too early.

But the fork/release is already deleted. Sorry for bothering you. I may start contributing if I find anything I can help with.
